flow/project.py

```diff
@@ -3169,12 +3309,33 @@ def next_operations(self, *jobs, ignore_conditions=IgnoreConditions.NONE):
         :type ignore_conditions:
             :py:class:`~.IgnoreConditions`
         :yield:
-            All instances of :class:`~.JobOperation` jobs are eligible for.
+            All instances of :class:`~._JobOperation` jobs are eligible for.
         """
         for job in jobs:
             for op in self._job_operations(job, ignore_conditions):
                 yield op
 
+    @deprecated(deprecated_in="0.10", removed_in="0.13", current_version=__version__)
+    def next_operations(self, *jobs, ignore_conditions=IgnoreConditions.NONE):
+        """Determine the next eligible operations for jobs.
+
+        :param jobs:
+            The signac job handles.
+        :type job:
+            :class:`~signac.contrib.job.Job`
+        :param ignore_conditions:
+            Specify if pre and/or post conditions check is to be ignored for eligibility check.
+            The default is :py:class:`IgnoreConditions.NONE`.
+        :type ignore_conditions:
+            :py:class:`~.IgnoreConditions`
+        :yield:
+            All instances of :class:`~.JobOperation` jobs are eligible for.
+        """
+        warnings.warn("The JobOperation class is deprecated as of 0.11 and "
+                      "will be removed in 0.13.",
+                      DeprecationWarning)
+        return self._next_operations(*jobs, ignore_conditions=ignore_conditions)
```

Comment:
We need to this to also transform `_JobOperation` objects into `JobOperation` objects. This should be doable as the necessary information is stored in the `_JobOperation` objects.

Reply:
This can be resolved by not returning `self._next_operations(arugements)` but rather we could copy the code and in `next_operations` we should yield JobOperation(op.name, op.id, op.job, ...)`

Reply:
That works too. I am fine with either solution.